staging/src/k8s.io/kubectl/go.mod

```diff
@@ -13,10 +13,9 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
 	github.com/fatih/camelcase v1.0.0
-	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/fvbommel/sortorder v1.0.1
 	github.com/go-openapi/spec v0.19.3
-	github.com/golangplus/testing v0.0.0-20180327235837-af21d9c3145e // indirect
```

Comment:
I'd like to underatand these changes too

Reply:
go.mod changes were do to the go version mismatch. Switching to go1.16 removed these changes.